blackbox/docs/ingestion.txt

```diff
@@ -0,0 +1,52 @@
+.. _ingestion:
+
+===================
+Ingestion Framework
+===================
+
+Aside from using standard SQL to ingest data, CrateDB offers the possibility to
+ingest data through custom event-based ingestion endpoints. The ingestion
+framework of CrateDB provides highly available and distributed endpoints for
```

Comment:
I think here we should elaborate a bit on what a ingestion endpoint is. And in exchange maybe get rid of the marketing speak below